CONTRIBUTING.rst

```diff
@@ -98,20 +97,15 @@ Pull Request Guidelines
 Before you submit a pull request, check that it meets these guidelines:
 
 1. The pull request should include tests.
-2. If the pull request adds functionality, the docs should be updated. Put
-   your new functionality into a function with a docstring, and add the
-   feature to the list in README.rst.
-3. The pull request should work for Python 3.8, 3.9 and 3.10, and for PyPy. Check
-   https://travis-ci.com/tialab/tiatoolbox/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+2. If the pull request adds functionality, the docs should be updated. Put your new functionality into a function with a docstring, and add the feature to the list in README.rst.
```

Comment:
What goes in the readme.rst? It's currently empty right?

Reply:
Good catch. I think it is not necessary. This should be deleted.